exercises/practice/word-count/.docs/instructions.md

```diff
@@ -12,7 +12,7 @@ When counting words you can assume the following rules:
 
 1. The count is _case insensitive_ (ie "You", "you", and "YOU" are 3 uses of the same word)
 2. The count is _unordered_; the tests will ignore how words and counts are ordered
-3. Other than the apostrophe in a _contraction_ all forms of _punctuation_ are ignored
+3. Other than the apostrophe in a _contraction_ all forms of _punctuation_ are regarded as spaces
```

Comment:
This should be a separate PR. I know that is a bit annoying but it is easier to review that way and a potential discussion about a completely different topic will not hold back the actual content from being merged. (Not saying there is anything to discuss here but in theory, that could be the case.)

Also remember that we squash commits so you don't have a separate history about the different changes in one PR as you would have otherwise.

Reply:
Thanks, will make another one in the future. Will wait until the last generator is merged (#2341) and then I will run all generators and everything again